features/mbedtls/mbed-crypto/importer/Makefile

```diff
@@ -29,7 +29,7 @@
 
 # Set the Mbed Crypto release to import (this can/should be edited before
 # import)
-CRYPTO_RELEASE ?= mbedcrypto-1.0.0d2
+CRYPTO_RELEASE ?= mbedcrypto-1.0.0d6
```

Comment:
did you mean mbedcrypto-1.0.0d5  or mbedcrypto-1.0.0d6 
 as in the headline?

Reply:
Updated headline to explain what we are doing now. Originally, this was just d5, now it is both d5 and then d6.